CONTRIBUTING.md

```diff
@@ -1,14 +1,69 @@
 # How to contribute
 
-## OBS Releases
+## OBS Packaging
 
-The CI will automatically interact with SUSE's [Open Build Service](https://build.opensuse.org): the master branch will be kept in sync with the `server:monitoring` project (the development upstream), while a new Submit Request against the `openSUSE:Factory` project (the stable downstream) can be triggered by publishing a new GitHub release or pushing a new Git tag.
- 
-When releasing to `openSUSE:Factory`, please ensure that tags always follow the [SemVer](https://semver.org/) scheme, and that [the changelog](prometheus-hanadb_exporter.changes) contains a new entry, otherwise the request submission might fail.
+The CI will automatically interact with SUSE's [Open Build Service](https://build.opensuse.org): the `master` branch will be kept in sync with the `network:ha-clustering:sap-deployments:devel` project.
+
+### Publishing the Exporter RPM in openSUSE code stream.
+
+For the exporter only, a new Submit Request against the `server:monitoring` project can be triggered by publishing a new GitHub release.
```

Comment:
Is `server:monitoring` still used?

Reply:
Yeah, the exporter packages released in `openSUSE:Factory` are still linked to `server:monitoring` as dev upstream.
I will propose to re-structure all our projects in OBS in the future.